packages/web/src/pages/upload-page/fields/AccessAndSaleField.tsx

```diff
@@ -169,7 +171,7 @@ export const AccessAndSaleFormSchema = (
     // Check for albumTrackPrice price >= min price (if applicable)
     .refine(
       (values) =>
-        isAlbum
+        isAlbum && isUpload
```

Comment:
i don't have full context here but don't we want to still validate the price even when it's not an upload?

Reply:
Yeah this is specifically the "track price" for all the tracks in the album. We don't have that field on edits now